lib/streamlit/config.py

```diff
@@ -977,13 +998,12 @@ def append_newline():
 # Load Config Files #
 
 
-# Indicates that this was defined by the user.
-_USER_DEFINED = "<user defined>"
-
-
 def _set_option(key, value, where_defined):
     """Set a config option by key / value pair.
 
+    This function assumes that the _config_options dictionary has already been
+    populated and thus should only be used within this file and by tests.
```

Comment:
Not sure if it's worth adding an assertion/check to make sure `_config_options` so we can get a nice error? 🤷

Reply:
Yeah it'd be good to add an `assert` here, will do